src/clp_logging/handlers.py

```diff
@@ -621,7 +644,7 @@ def _write(self, loglevel: int, msg: str) -> None:
         try:
             if self.closed:
                 raise RuntimeError("Socket already closed")
-            clp_msg: bytearray = CLPEncoder.encode_message(msg.encode())
+            clp_msg: bytearray = FourByteEncoder.encode_message((msg + "\n").encode())
```

Comment:
Same as above. Copied for reference:

Lets move the `+ "\n"` to CLPBaseHandler.emit. This will avoid making sure we need to add it in each `_write` path.

(It probably should have been there in the original version as well.)